src/webview/create-component/createComponentLoader.ts

```diff
@@ -530,6 +539,9 @@ function clone(url: string, location: string, branch?: string): Promise<ClonePro
     const git = gitExtension.getAPI(1).git.path;
     let command = `${git} clone ${url} ${location}`;
     command = branch ? `${command} --branch ${branch}` : command;
```

Comment:
Ask Roland about this, there should be a way to get json output from git clone so that you can get precise progress numbers. If you run git using `spawn` from `node:child_process`, then you should be able to attach a callback that runs whenever something is printed to the terminal, and you can use this to send a message to the webview that updates the progress.

Reply:
The progress is reported onto `stderr` if you pass the `--progress` option to `git`, I asked Roland and it seems there is not a nice way to get JSON.

Reply:
Here the percentage of completion not only for clone, It was from clone to analyze flow. So I segregated the completion percentage on each level.

Reply:
Okay, this makes sense to me.